data/templates/dhcp-server/dhcpd.conf.tmpl

```diff
@@ -165,7 +165,7 @@ shared-network {{ network | replace('_','-') }} {
         option wpad-url "{{ subnet_config.wpad_url }}";
 {%         endif %}
 {%         if subnet_config.client_prefix_length is defined and subnet_config.client_prefix_length is not none %}
-        option subnet-mask {{ subnet_config.client_prefix_length }};
+        option subnet-mask {{ subnet_config.client_prefix_mask }};
```

Comment:
This PR looks a bit overenginnered. Why not use one of our helpers?

```
option subnet-mask {{ subnet_config.client_prefix_length | netmask_from_cidr }};
```

https://github.com/vyos/vyos-1x/blob/current/python/vyos/template.py#L165-L173

Reply:
Thanks @c-po ,
Your version does look much more attractive
I'll keep the existing helpers in mind
fixed!